lang/en-us.json

```diff
@@ -213,8 +213,8 @@
             "npc_dota_hero_winter_wyvern": "Winter Wyvern",
             "npc_dota_hero_witch_doctor": "Witch Doctor",
             "npc_dota_hero_skeleton_king": "Wraith King",
-            "npc_dota_hero_skeleton_king_2": "Skeleton King",
-            "npc_dota_hero_skeleton_king_3": "Skeleton King",
+            "npc_dota_hero_skeleton_king_2": "Wraith King",
```

Comment:
You found my easter egg and fixed it 😆

Reply:
Oh, that was an easter egg XD.
I thought you had taken the hero name from something and was left with incorrect information.